test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/CSharpReservedWordsTest.cs

```diff
@@ -7,6 +7,11 @@ namespace Microsoft.AspNetCore.Razor.Language.Legacy
 {
     public class CSharpReservedWordsTest : CsHtmlCodeParserTestBase
     {
+        public CSharpReservedWordsTest()
+        {
+            UseBaselineTests = true;
```

Comment:
I added this temporary flag to help convert these parser tests one by one instead of all at once.